src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToLocalizedStringTransformer.php

```diff
@@ -163,6 +163,10 @@ protected function getIntlDateFormatter($ignoreTimezone = false)
         $dateFormat = $this->dateFormat;
         $timeFormat = $this->timeFormat;
         $timezone = $ignoreTimezone ? 'UTC' : $this->outputTimezone;
+        if (!defined('HHVM_VERSION') && PHP_VERSION_ID >= 50500) {
```

Comment:
It looks like this could enhance portability:
```
if (class_exists('IntlTimeZone', false)) {
    $timezone = IntlTimeZone::createTimeZone($timezone)
}
```